enterprise/static/enterprise/sass/partials/views/_data_sharing_consent.scss

```diff
@@ -170,3 +107,11 @@
     }
   }
 }
+
+@media (max-width: map-get($container-max-widths, md)) {
+  .enterprise-container .row {
+    [class*='col-'] {
+      max-width: 100%;
+    }
+  }
+}
```

Comment:
nit: new line at end of file

Also curious why it was necessary to override the `col-*` classes' `max-width` property?